lsp/integration/project-test/.civet/coffee-plugin.mjs

```diff
@@ -0,0 +1,58 @@
+// Rudimentary CoffeeScript plugin
```

Comment:
Is there some way for the end-user to use this? Perhaps it could be released as a second VSCode plugin, and users wanting both could install both?

Reply:
Copying this file to the `.civet` folder should enable CoffeeScript integration for the LSP the same way that `.hera` is supported in the root of the Civet repo.

Reply:
Can we document this? I had no idea about either hack... Though now I see the general concept is in the README! Perhaps we could add a specific link to `coffee-plugin` from the README?